packages/terra-status-view/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ Changelog
 
 Unreleased
 ----------
+### Changed
+* Updated status view variants object, added proptype check
```

Comment:
This might be worth splitting into a couple different entries when resolving the conflicts. 

Something similar to this

### Breaking Change
* Removed `StatusView.Opts` export

### Added
* Added additional variants export

Reply:
Changes made! [fd9c33d](https://github.com/cerner/terra-core/pull/2264/commits/fd9c33d8e3034b0c78d362541b68921ca293de36)